src/api/api.js

```diff
@@ -27,7 +25,7 @@ async function createEvent(event) {
 
   let res;
   try {
-    res = await protectedResourceAxios.post(`${baseUrl}/protected/events/`, body);
+    res = await protectedResourceAxios.post(`${process.env.VUE_APP_API_DOMAIN}/api/v1/protected/events/`, body);
```

Comment:
Can you keep the baseUrl const and just set it to this? I think that's easier to read than using this variable directly

Reply:
We can build out a more complete structure when we have more routes, we shouldn't just have a const above.